app/src/features/auth/user.ts

```diff
@@ -0,0 +1,24 @@
+import { createAction, createSlice } from "@reduxjs/toolkit";
```

Comment:
maybe we should follow the file naming convention here. How about userSlice.ts ? Don't know if that's necessary..